### PR TITLE
Node8 upgrade

### PR DESCRIPTION
**Pull request by hvhallmann**
update harvester reference, to use the last one
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/agco/elastic-harvesterjs/pull/94%23issuecomment-324131784%22%2C%20%22https%3A//github.com/agco/elastic-harvesterjs/pull/94%23issuecomment-324145057%22%2C%20%22https%3A//github.com/agco/elastic-harvesterjs/pull/94%23issuecomment-333536326%22%2C%20%22https%3A//github.com/agco/elastic-harvesterjs/pull/94%23discussion_r136089456%22%2C%20%22https%3A//github.com/agco/elastic-harvesterjs/pull/94%23discussion_r136090075%22%2C%20%22https%3A//github.com/agco/elastic-harvesterjs/pull/94%23discussion_r136096192%22%2C%20%22https%3A//github.com/agco/elastic-harvesterjs/pull/94%23discussion_r136096770%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/agco/elastic-harvesterjs/pull/94%23issuecomment-324131784%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5Cn%5B%21%5BCoverage%20Status%5D%28https%3A//coveralls.io/builds/12944188/badge%29%5D%28https%3A//coveralls.io/builds/12944188%29%5Cn%5CnCoverage%20remained%20the%20same%20at%2077.514%25%20when%20pulling%20%2A%2A314737b67ccd0abb51c9967766b5e9275054e1c5%20on%20node8-upgrade%2A%2A%20into%20%2A%2A0a6f95d99793769bf5c9e4d885e60428c8be316c%20on%20develop%2A%2A.%5Cn%22%2C%20%22created_at%22%3A%20%222017-08-22T19%3A45%3A54Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/2354108%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/coveralls%22%7D%7D%2C%20%7B%22body%22%3A%20%22%5Cn%5B%21%5BCoverage%20Status%5D%28https%3A//coveralls.io/builds/12944975/badge%29%5D%28https%3A//coveralls.io/builds/12944975%29%5Cn%5CnCoverage%20remained%20the%20same%20at%2077.514%25%20when%20pulling%20%2A%2A8733bd50162b2890cdd0a2afa25477e549441dde%20on%20node8-upgrade%2A%2A%20into%20%2A%2A0a6f95d99793769bf5c9e4d885e60428c8be316c%20on%20develop%2A%2A.%5Cn%22%2C%20%22created_at%22%3A%20%222017-08-22T20%3A39%3A01Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/2354108%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/coveralls%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222017-10-02T13%3A36%3A48Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/7826016%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/HarielGiacomuzzi%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%206242ee45925906b6ce1be36332a89d008516a3fc%20package.json%2014%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/agco/elastic-harvesterjs/pull/94%23discussion_r136089456%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Better%20to%20use%20a%20harvester%20tag.%22%2C%20%22created_at%22%3A%20%222017-08-30T14%3A45%3A10Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/372075%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ssebro%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20need%20permission%20on%20npm%5Cr%5Cncan%20you%20provide%20it%2C%20please%3F%5Cr%5Cnhttps%3A//www.npmjs.com/package/harvesterjs%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222017-08-30T15%3A06%3A43Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/7022648%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hvhallmann%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20package.json%3AL16-23%22%7D%2C%20%22Pull%208733bd50162b2890cdd0a2afa25477e549441dde%20test/aggregations.spec.js%206%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/agco/elastic-harvesterjs/pull/94%23discussion_r136090075%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Why%20is%20this%20test%20skipped%20and%20not%20removed%2C%20if%20it%27s%20no%20longer%20important%3F%22%2C%20%22created_at%22%3A%20%222017-08-30T14%3A47%3A16Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/372075%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ssebro%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%27m%20not%20sure%20if%20it%20is%20not%20important%5Cr%5CnI%20got%20a%20lot%20of%20warning%20about%20the%20delete%20feature%20of%20elastic%20search%3B%5Cr%5CnSo%2C%20I%20believe%20it%20needs%20an%20investigation.%22%2C%20%22created_at%22%3A%20%222017-08-30T15%3A08%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/7022648%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hvhallmann%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20test/aggregations.spec.js%3AL227-240%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/HarielGiacomuzzi%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/7826016%3Fv%3D4%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/HarielGiacomuzzi'></a>

- [x] <a href='#crh-comment-Pull 6242ee45925906b6ce1be36332a89d008516a3fc package.json 14'></a> &nbsp;<b><a href='https://github.com/agco/elastic-harvesterjs/pull/94#discussion_r136089456'>File: package.json:L16-23</a></b>
- <a href='https://github.com/ssebro'></a> Better to use a harvester tag.
- <a href='https://github.com/hvhallmann'></a> I need permission on npm
can you provide it, please?
https://www.npmjs.com/package/harvesterjs
- [x] <a href='#crh-comment-Pull 8733bd50162b2890cdd0a2afa25477e549441dde test/aggregations.spec.js 6'></a> &nbsp;<b><a href='https://github.com/agco/elastic-harvesterjs/pull/94#discussion_r136090075'>File: test/aggregations.spec.js:L227-240</a></b>
- <a href='https://github.com/ssebro'></a> Why is this test skipped and not removed, if it's no longer important?
- <a href='https://github.com/hvhallmann'></a> I'm not sure if it is not important
I got a lot of warning about the delete feature of elastic search;
So, I believe it needs an investigation.
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/agco/elastic-harvesterjs/pull/94#issuecomment-324131784'>General Comment</a></b>
- <a href='https://github.com/coveralls'></a> [Coverage Status](https://coveralls.io/builds/12944188)
Coverage remained the same at 77.514% when pulling **314737b67ccd0abb51c9967766b5e9275054e1c5 on node8-upgrade** into **0a6f95d99793769bf5c9e4d885e60428c8be316c on develop**.
- <a href='https://github.com/coveralls'></a> [Coverage Status](https://coveralls.io/builds/12944975)
Coverage remained the same at 77.514% when pulling **8733bd50162b2890cdd0a2afa25477e549441dde on node8-upgrade** into **0a6f95d99793769bf5c9e4d885e60428c8be316c on develop**.

<a href='https://www.codereviewhub.com/agco/elastic-harvesterjs/pull/94?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/agco/elastic-harvesterjs/pull/94?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/agco/elastic-harvesterjs/pull/94?approve=0'></a>&nbsp;<a href='https://github.com/agco/elastic-harvesterjs/pull/94'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by ssebro** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/agco/elastic-harvesterjs/pull/94'>Refresh</a>.<a href='#crh-update-none'></a>

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "bluebird": "~2.3.2",
-    "harvesterjs": "2.2.1",
+    "harvesterjs": "git+https://github.com/agco/harvesterjs.git#99fd68e9189db80ac7861b0f878516e616856058",
```
Comment (ssebro) [inline package.json:19]: Better to use a harvester tag.

File: test/aggregations.spec.js [hunk 2]
```diff
@@ -227,10 +227,13 @@ describe('aggregations', function () {
             });
         });
 
-        it('should be able to remove linked documents', function (done) {
+        // I believe you should use delete
+        it.skip('should be able to remove linked documents', function (done) {
```
Comment (ssebro) [inline test/aggregations.spec.js:231]: Why is this test skipped and not removed, if it's no longer important?

File: package.json (hunk 1 above)
Comment (hvhallmann) [reply package.json:19]: I need permission on npm
can you provide it, please?
https://www.npmjs.com/package/harvesterjs

File: test/aggregations.spec.js (hunk 2 above)
Comment (hvhallmann) [reply test/aggregations.spec.js:231]: I'm not sure if it is not important
I got a lot of warning about the delete feature of elastic search;
So, I believe it needs an investigation.

**Comment by HarielGiacomuzzi** [comment]
:shipit:

**Pull request merged**